### PR TITLE
feat: add github actions bot to handle PRs #29

### LINKED ISSUE #29: feature: add github actions bot to handle PRs

### PR DESCRIPTION
**Pull request by regevbr**
Fix #29

### CODE REVIEW

**Comment by regevbr** [comment]
@ynahmany can you please review?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -24,12 +24,16 @@ jobs:
...
+        node-version: [10.x, 12.x, 13.x]
         exclude:
           - os: macos-latest
             node-version: 10.x
+          - os: macos-latest
+            node-version: 13.x
           - os: windows-latest
             node-version: 10.x
+          - os: windows-latest
+            node-version: 13.x
```
Comment (ynahmany) [inline .github/workflows/ci.yml]: why node 13.x ?

File: package.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
   },
   "engineStrict": true,
   "engines": {
-    "node": "^10 || ^12"
+    "node": "^10 || ^12 || ^13"
```
Comment (ynahmany) [inline package.json]: how come you added 13? it is not stable..

File: .github/workflows/ci.yml (hunk 1 above)
Comment (regevbr) [reply .github/workflows/ci.yml]: This is an exclude - mac and windows are only tested on node 12

File: package.json (hunk 2 above)
Comment (regevbr) [reply package.json]: It is indeed not stable, but it is the base ground for node 14 and I think we should be already prepared for node 14 when it comes out in a month or so. When node 14 will be out, we will remove support for node 13.
In addition, if someone does use node 13, why should we not support it?
Thoughts?

Comment (ynahmany) [reply package.json]: I haven't read the spec for node 14 so I am not aware for the big changes yet.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ynahmany) [reply .github/workflows/ci.yml]: sorry, missed that.

**Pull request merged**